tracer/src/Datadog.Trace/Configuration/DynamicConfigurationManager.cs

```diff
@@ -76,6 +76,7 @@ private static void OnConfigurationChanged(ConfigurationBuilder settings)
                 LogsInjectionEnabled = settings.WithKeys(ConfigurationKeys.LogsInjectionEnabled).AsBool(),
                 HeaderTags = headerTags == null ? null : new ReadOnlyDictionary<string, string>(headerTags),
                 // ServiceNameMappings = serviceNameMappings == null ? null : new ReadOnlyDictionary<string, string>(serviceNameMappings)
+                GlobalTags = globalTags == null ? null : new ReadOnlyDictionary<string, string>(globalTags)
```

Comment:
Feels like there should be a `ReadOnlyDictionary.Empty<T1, T2>()`! 😄

Reply:
Is the idea here that if we don't specify any tags here, this will return null so we ignore it and use the local configuration? Is there an expected behavior if it is set and then a user explicitly removes the config from dynamic configuration?

Reply:
> Is the idea here that if we don't specify any tags here, this will return null so we ignore it and use the local configuration?

Correct. Every key in dynamic config is optional.

> Is there an expected behavior if it is set and then a user explicitly removes the config from dynamic configuration?

When that happens, we should restore the old value (like if the user never set any value in dynamic config)